bakery/tasks/t-fetch-book.yml

```diff
@@ -17,5 +17,5 @@ run:
     - -cxe
     - |
       exec 2> >(tee book/stderr >&2)
-      neb get -r -d "./temp" "staging" "$(cat book-title/collection_id)" "$(cat book-title/version)"
+      neb get -r -d "./temp" "$(cat book-title/server | cut -f1 -d".")" "$(cat book-title/collection_id)" "$(cat book-title/version)"
```

Comment:
This only happens to work because the neb default config uses this convention. Should that ever change, this would stop working. Is it worth generating a neb_config file as part of the task?

Reply:
that's probably good thing to do after this. it doesn't necessarily need to be in this PR if this is functional at the time. it can be a task added for getting this ready for production.

Reply:
yes, this was a bit of hackery. I can create a card to address on production ready epic.

Reply:
Issue created to capture and make proper updates - https://github.com/openstax/cnx/issues/782